Sail/src/Sail/entities/systems/network/NetworkSenderSystem.cpp

```diff
@@ -19,12 +19,16 @@
 #include <string>
 #include <vector>
 
+
+#include "Sail/../../libraries/gzip/compress.hpp"
```

Comment:
Should perhaps be added to include libraries instead?

Reply:
libraries is already in the included folders so just "gzip/compress.hpp" should work